builds/setting-up-trusted-ca.adoc

```diff
@@ -0,0 +1,26 @@
+[id="setting-up-trusted-ca"]
+= Setting up additional trusted certifying authorities for builds
+include::modules/common-attributes.adoc[]
+:context: setting-up-trusted-ca
+toc::[]
+
+Use the following sections to set up additional certifying authorities (CA).
+
+In general, a cluster admin creates a ConfigMap and adds additional CAs.
+
+* No longer needs single PEM bundle
```

Comment:
What no longer needs a single PEM bundle? I would make this a complete sentence and add a period, to keep consistent with the rest of the list.